src/CppParser/Parser.cpp

```diff
@@ -903,6 +903,14 @@ void Parser::WalkRecord(const clang::RecordDecl* Record, Class* RC)
         ReadClassLayout(RC, Record, CharUnits(), true);
     }
 
+    if (Record->redecls_begin() != Record->redecls_end()) {
+        for (auto redecl : Record->redecls()) {
+            auto dec = WalkDeclaration(redecl);
+            if (dec != nullptr)
```

Comment:
I don't think it's possible for the returned result to be null so remove the check unless it really is necessary.